khronicle-core/src/commonMain/kotlin/SimpleLogger.kt

```diff
@@ -0,0 +1,31 @@
+package com.juul.khronicle
+
+/** Simplified [Logger] which can log all */
```

Comment:
nit: Missing period.

```suggestion
/** Simplified [Logger] which can log all. */
```